<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDockWidget</class>
 <widget class="QDockWidget" name="FindDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>55</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="searchBox">
      <property name="placeholderText">
       <string>search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btnPrev">
      <property name="text">
       <string>Previous</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::LeftArrow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btnNext">
      <property name="text">
       <string>Next</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="casesensitive">
      <property name="text">
       <string>Casesensitive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="fulldoc">
      <property name="text">
       <string>Full document</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
